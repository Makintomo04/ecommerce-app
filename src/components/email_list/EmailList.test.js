import React from "react"
import {shallow} from "enzyme"
import EmailList from "./EmailList"


it("Should render Blog component correctly", () => {
    const emailList = shallow(<EmailList />);
    expect(emailList).toMatchSnapshot(); // check if a component renders correctly
  });