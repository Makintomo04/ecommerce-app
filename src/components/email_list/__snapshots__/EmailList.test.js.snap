// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render EmailList component correctly 1`] = `
<div
  className="email-list"
>
  <span
    className="email-list__text"
  >
    SIGN UP TO RECEIVE 15% OFF YOUR FIRST ORDER
  </span>
  <input
    className="email-list__input"
    id="email"
    name="email"
    placeholder="Enter email address"
    type="text"
  />
  <input
    className="email-list__btn"
    type="submit"
    value="SIGN UP"
  />
</div>
`;
